apps/qubit/modules/informationobject/templates/_renameModal.php

```diff
@@ -9,45 +9,32 @@
 
   <form id="renameModalForm" action="<?php echo url_for(array('module' => 'informationobject', 'action' => 'rename', 'slug' => $resource->slug)) ?>" method="POST">
 
-  <div class="modal-body">
-    <div class="alert"><?php echo __('Use this interface to update the description title, slug (permalink), and/or digital object filename.') ?></div>
+    <div class="modal-body">
+      <div class="alert"><?php echo __('Use this interface to update the description title, slug (permalink), and/or digital object filename.') ?></div>
 
-    <div>
       <div style="float:right"><input id="rename_enable_title" type="checkbox" checked="checked" /> <?php echo __('Update title') ?></div>
```

Comment:
Can you avoid inline CSS if possible? Also, for id and class attributes we normally use dashes (e.g. rename-enable-title). You may find some places where camelCase is used, but that's being changed.